scrape/scrape.go

```diff
@@ -1512,7 +1512,8 @@ loop:
 
 		t := defTime
 		if isHistogram {
-			met, parsedTimestamp, h = p.Histogram()
+			met, parsedTimestamp, h, _ = p.Histogram()
+			// TODO: ingest float histograms in tsdb
```

Comment:
Extremely small nit. We follow this comment style in Prometheus.
```suggestion
			// TODO: ingest float histograms in tsdb.
```